Octokit/Models/Request/PublicRepositoryRequest.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Octokit
+{
+    [DebuggerDisplay("{DebuggerDisplay,nq}")]
+    public class PublicRepositoryRequest : RequestParameters
+    {
+        public PublicRepositoryRequest()
+        {
+        }
+
+        public long Since { get; set; }
```

Comment:
yep, certainly as if you wanted to find **ALL** public repos you'd just call the parameterless `GetAllPublic` method :see_no_evil: